game-core/src/main/java/games/strategy/triplea/ui/PoliticalStateOverview.java

```diff
@@ -43,36 +46,45 @@ public PoliticalStateOverview(
     drawPoliticsUi();
   }
 
+  private void addCell(final Component cell, final Insets insets, final int x, final int y) {
```

Comment:
Code nitpick: Please check the method ordering convention:
https://github.com/triplea-game/triplea/wiki/Code-Conventions#depth-first-method-ordering

We want to avoid new methods being added *above* where they are used. Specifically here "addCell" should be below the method that first uses it.